modules/ui/src/main/scala/lucuma/ui/components/CopyTextToClipboard.scala

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui.components
+
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.vdom.html_<^.*
+import lucuma.react.common.*
+import org.scalajs.dom.window.navigator
+
+type OnCopy = (String, Boolean) => Callback
+
+/**
+ * Text to be copied to clipboard Optional callback, will be called when text is copied Optional
+ * copy-to-clipboard options.
```

Comment:
Comment could be improved. Did it get auto-combined onto one line?
A description of the parameters to 'onCopy` would be useful.